### ISSUE TITLE
Uncaught TypeError: path must be a string

### ISSUE THREAD

**Issue by chris-aeviator** [dingjie/atom-sync#19]
1. Installed atom-sync via apm
2. Restarted Atom
3. Disabled remote-sync plugin
4. Switched to project and right clicked folder , sync -> edit sync config

**Atom Version**: 1.3.2
**System**: Mac OS X 10.10.5
**Thrown From**: [atom-sync](https://github.com/dingjie/atom-sync) package, v0.5.2

### Stack Trace

Uncaught TypeError: path must be a string

```
At fs.js:549

TypeError: path must be a string
    at TypeError (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.module.(anonymous function) [as openSync] (ATOM_SHELL_ASAR.js:137:20)
    at Object.fs.writeFileSync (fs.js:1171:15)
    at Object.fsPlus.writeFileSync (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fs-plus/lib/fs-plus.js:279:17)
    at Object.module.exports.ConfigHelper.initialise (/Users/korny/.atom/packages/atom-sync/lib/helper/config-helper.coffee:10:16)
    at Object.module.exports.ServiceController.onCreate (/Users/korny/.atom/packages/atom-sync/lib/controller/service-controller.coffee:23:17)
    at HTMLDivElement.subscriptions.add.atom.commands.add.atom-sync:configure (/Users/korny/.atom/packages/atom-sync/lib/atom-sync.coffee:12:25)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:160:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:1016:28)
    at emitTwo (events.js:87:13)
    at EventEmitter.emit (events.js:172:7)
    at EventEmitter.ipc.sendSync (/Applications/Atom.app/Contents/Resources/atom.asar/renderer/api/lib/ipc.js:21:31)
    at BrowserWindow.RemoteMemberFunction [as emit] (/Applications/Atom.app/Contents/Resources/atom.asar/renderer/api/lib/remote.js:143:29)
    at ContextMenuManager.module.exports.ContextMenuManager.showForEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/context-menu-manager.js:156:33)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentContextmenu (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:250:47)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -0:02.7.0 atom-sync:configure (span.name.icon.icon-repo)
```

### Config

```json
{
  "core": {
    "themes": [
      "predawn-ui",
      "twilight-syntax-spacegray-eighties"
    ],
    "disabledPackages": [
      "wrap-guide",
      "linter-rails-best-practices",
      "linter-pep8",
      "pigments",
      "linter-php",
      "remote-sync"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-jinja2, v0.5.0
atom-sync, v0.5.2
bookmarks, v0.38.0
editorconfig, v1.2.2
emmet, v2.3.14
file-icons, v1.6.11
jshint, v1.8.3
language-sass, v0.45.0
linter, v1.11.2
linter-csslint, v1.1.0
linter-erb, v0.1.1
linter-jshint, v1.2.1
linter-jslint, v10.0.0
linter-jsonlint, v1.0.1
linter-phpmd, v1.3.2
linter-tidy, v1.0.1
predawn-ui, v1.0.2
project-manager, v2.6.5
sass-autocompile, v0.11.0
terminal-plus, v0.14.5
twilight-syntax-spacegray-eighties, v0.3.0

# Dev
settings-view, v0.224.0
```

**Comment by dingjie** [comment]
Would you try if it's been resolved in [0.6.0](https://github.com/dingjie/atom-sync/releases/tag/v0.6.0) ?

**Comment by chris-aeviator** [comment]
I can confirm it's resolved,
Thanks for taking action!
